projects/commudle-admin/src/app/components/home/communities/home-community-card/home-community-card.component.scss

```diff
@@ -48,3 +48,55 @@
     }
   }
 }
+
+
+@media only screen and (max-width: 768px){
```

Comment:
please use the max width from variables.scss (import the variable and use it). We want to show this layout only on the mobile width and smaller screen sizes for now